bundle/manifests/gpu-operator-certified.clusterserviceversion.yaml

```diff
@@ -201,37 +201,37 @@ spec:
     - name: gpu-operator-image
       image: ghcr.io/nvidia/gpu-operator:main-latest
```

Comment:
What happens to this image when we add the commit SHA as tag in #1898? Will it be ignored?